src/screens/Analytics/Logs/LogUtils/AuditLogUI.res

```diff
@@ -0,0 +1,210 @@
+      ->Dict.toArray
+      ->Array.filter(item => {
+        let (key, value) = item
+        !(LogUtils.detailsSectionFilterKeys->Array.includes(key)) && value->isValidNonEmptyValue
+      })
+      ->Array.map(item => {
+        let (key, value) = item
+        <div className="text-sm font-medium text-gray-700 flex">
+          <span className="w-2/5"> {key->snakeToTitle->React.string} </span>
+          <span
+            className="w-3/5 overflow-scroll cursor-pointer relative hover:bg-gray-50 p-1 rounded">
+            <div
+              onClick={_ => handleOnClickCopy(~parsedValue=value->JSON.stringify)}
+              className="w-full h-full absolute top-0 flex justify-end opacity-0 hover:opacity-100 p-1">
+              <div className="cursor-pointer">
+                <Icon name="copy-code" />
+              </div>
+            </div>
+            <ReactSyntaxHighlighter.SyntaxHighlighter
+              wrapLines={true}
+              style={ReactSyntaxHighlighter.lightfair}
+              language="json"
+              showLineNumbers={false}
+              lineNumberContainerStyle={{
+                paddingLeft: "0px",
+                backgroundColor: "red",
+                padding: "0px",
+              }}
+              customStyle={{
+                backgroundColor: "transparent",
+                fontSize: "0.875rem",
+                padding: "0px",
+              }}>
+              {value->JSON.stringify}
+            </ReactSyntaxHighlighter.SyntaxHighlighter>
+          </span>
+        </div>
+      })
+      ->React.array}
+    </div>
+  }
+}
+
+@react.component
+let make = (~id, ~promiseArr, ~logType: LogTypes.pageType) => {
+  open LogicUtils
+  open LogUtils
+  open LogTypes
+  let (data, setData) = React.useState(_ => [])
+  let isError = React.useMemo0(() => {ref(false)})
+  let (logDetails, setLogDetails) = React.useState(_ => {
+    response: "",
+    request: "",
+    data: Dict.make(),
+  })
+  let (selectedOption, setSelectedOption) = React.useState(_ => {
+    value: 0,
+    optionType: API_EVENTS,
+  })
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
+
+  let getDetails = async () => {
+    let logs = []
+
+    if !(id->HSwitchOrderUtils.isTestData) {
+      let resArr = await PromiseUtils.allSettledPolyfill(promiseArr)
+
+      resArr->Array.forEach(json => {
+        // clasify btw json value and error response
+        switch JSON.Classify.classify(json) {
+        | Array(arr) =>
+          // add to the logs only if array is non empty
+          switch arr->Array.get(0) {
+          | Some(dict) =>
+            switch dict->getDictFromJsonObject->getLogType {
+            | SDK => logs->Array.pushMany(arr->parseSdkResponse)->ignore
+            | CONNECTOR | API_EVENTS => logs->Array.pushMany(arr)->ignore
+            | WEBHOOKS => logs->Array.pushMany([dict])->ignore
+            }
+          | _ => ()
+          }
+        | String(_) => isError.contents = true
```

Comment:
please recheck this.Even if single value is string your setting isError has true.